components/esp_websocket_client/examples/pytest_websocket.py

```diff
@@ -65,19 +63,19 @@ def __exit__(self, exc_type, exc_value, traceback):
         if self.thread.is_alive():
             Utility.console_log('Thread cannot be joined', 'orange')
 
-
+@pytest.mark.skip()
```

Comment:
Does it mean that this test is skipped?
Or is this needed for mark an internal function, so `pytest` doesn't treat it as an official test function? Could we make it an internal function if so? (e.g. move it to `test_examples_protocol_websocket()` scope)